### PR TITLE
feat: Remove Zipkin dependency

### PR DESCRIPTION
**Pull request by alainbodiguel**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -55,30 +55,29 @@
...
-    <zipkin.dropwizard.version>2.0.16-1</zipkin.dropwizard.version>
-    <swaggerv3.version>2.2.3</swaggerv3.version>
+    <swaggerv3.version>2.2.6</swaggerv3.version>
     <jaxb.version>2.3.6</jaxb.version>
     <jersey-media-jaxb.version>2.37</jersey-media-jaxb.version>
     <activation.version>2.1.0</activation.version>
 
     <!-- CACHE-->
-    <hazelcast.version>5.1.4</hazelcast.version>
+    <hazelcast.version>5.2.0</hazelcast.version>
```
Comment (sfalquier) [inline pom.xml:64]: Shouldn't these dependencies upgrades be merged in another commit/PR ?

**Review by sfalquier — APPROVED** [review]
LGTM

**Review by sebbousquet — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (alainbodiguel) [reply pom.xml:64]: Indeed!
